src/server/system_services/system_server.js

```diff
@@ -504,9 +509,8 @@ function read_system(req) {
         //     dbg.log0('read_system: n2n_config.stun_servers', n2n_config.stun_servers);
         // }
 
-        let last_upgrade = system.last_upgrade && {
-            timestamp: system.last_upgrade.timestamp,
-            last_initiator_email: system.last_upgrade.initiator
+        let last_upgrade = system.upgrade_history.successful_upgrades[0] && {
+            timestamp: system.upgrade_history.successful_upgrades[0].timestamp
```

Comment:
We should probably remove from the FE the initiator as well or we'll fail on schema

Reply:
for now, I tried to reduce any changes in the FE. I filled in any fields required by the UI to not break it